exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/QueryStateDisplayName.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.server.rest.profile;
+
+import org.apache.drill.exec.proto.UserBitShared.QueryResult.QueryState;
+
+public class QueryStateDisplayName {
+  // Values should correspond to the QueryState enum in UserBitShared.proto
```

Comment:
Not quite right, value do not correspond to QueryState enum, it's rather a mapping to QueryState enum. To be more clear here I suggest you create a map with `QueryState` as a key and `String` as value:
```
STARTING -> Starting
COMPLETED -> Succeeded
```
Thus you'll be able to get desired representation value by `QueryState` and in case if value is null return some default, like `Unknown State`.

Reply:
QueryState is an enum.
I feel having an array is simple. We could use the QueryState ordinal as index to fetch the display name from the array. I can add a check to check for array index overflow, and return some default value like "Unknown state".

Reply:
@prasadns14 
1. When using array you depend on enum value order. If for some reason value order changes, your approach will be working incorrectly.
2. Having map will show which key is mapped to which value and it will be much easier to understand that, for example, completed is mapped to succeeded.

I recommend you use map approach.

Reply:
@arina-ielchiieva 
yes, map will definitely make it to easier to visualize the mapping. 
Made the changes